### DISCUSSION TITLE
Puppy Linux 421 t8 work confirmed (texlive 2008 mod chu, by vanchutr)

### DISCUSSION THREAD

**Discussion by proxy-m** [copy/v86#1348] [Show and tell]
Puppy Linux pup421t8 work confirmed (old pup421 + t8 modification, vietnamese users, by vanchutr).

Source: [from murga linux forum archive](https://oldforum.puppylinux.com/viewtopic.php?t=52913)

HOW TO run under V86: 
-----------------------------------------------------
1. Disable network (optional, because not maintainer).
2. Basic sound works in urxvt (via speaker js).
3. Wait grub4dos menu start (1 sec, `root (cd)` fix was ).
4. Enter on Chupup-431 NO-SFS RAM (or wait +10 sec)
5. Wait loading main sfs, kernel modules, modules and services (+2 min).
6. Several enter on choose mouse & keyboard.
7. Enter Xorg or Xvesa (x11 gui).
8. Several enter for Xorg default params (+3 min).
9. After x11 programs autorun (+9 min).
10. Total time to boot to gui desktop: ~15 min.

CHANGES to Puppy 421 on chu tex2008: 
-----------------------------------------------------

1. Add: Link_SFS is used for hot-load some .sfs files. 
We can load and use immediately these .sfs (and more) files: 
adobe-9.3-1-sfs4-sfs3.sfs 
devx_431-sfs3.sfs 
imamagick.sfs 
k3b105.sfs 
minioo3-sfs3.sfs 
wine-1.1.38-sfs3.sfs 

and many other .sfs files 

2. Unikey+XVnkb (Vietnamse Input Method for vietnamese users). 

3. GhostScript 8.70 (to render pdf file more efficient) 

4. TeXMaker: (TeX Editor - Small) 
Start from console -> Use command: # texmaker 

5. TeXLive 2008 Mini = VnTeX, OmegaVn 
Mini-installation => Some packages will be missed, but they can be added easily (Copy the missed packages to work directory). 
Add VnTeX and OmegaVN for vietnamese people 

6. Djview -> Run from desktop shortcut. 
7. Shot-screen -> Run from desktop shortcut (Snapp screen or windows) 
8. You2pup -> Run from desktop shortcut 
9. Add wl.ko for BroadComp Wifi driver. Tested with T43 IBM ThinkPad (old laptop) and Emachines T725 (new and cheap laptop). 
10. Geany editor: Old version (0.13) with vietnamse interface. In case, you don't like viet-interface you can delete the geany.mo in /usr/local/share/local/vi/LC_MESSAGES directory and restart geany. 

11. Remove Abiword, disable puppyPDF.
11. Remove All softs in Fun Directory. 
12. rxvt is replaced by urxvt.
```

Screenshort [img1](https://i.ibb.co/zPZbDS7/image.png)
https://ibb.co/Yfh8wty

**Comment by proxy-m** [comment]
Image attached: https://www.dropbox.com/s/6r9o9aqv0fq9o4m/puppy421tex2008_modchu.iso?dl=1

md5sum: 2f160def7233a3c9f70d97f7dfb77373

sha256sum: db72411b7521f7da8beb4229513c37786940d72bc9cf7154f7a3e8af58b0cb4f  

sha1sum: 0a7928c74daba42ecd0a8f30bd5c21c5619d2cad

P.S.: Image published under no warranty, no limits, public domain, use as is, mit, or bsd:

```
THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

P.P.S.: Image content licences: puppy linux usually free, but maintainer of mod is not known.

**Comment by proxy-m** [comment]
Linux kernel: 2.6.25 i686 (known for puppy 420-421)
Loader: grub4dos 0.4.3 2007-10-15 (known grub 1 fork, not confuse with chenall grub4dos 0.4.5c 2013)
X11: Xorg/Xvesa (both possible).

**Comment by Pixelsuft** [comment]
I'm don't know how, but:
xenialpup-7.5-uefi works, but bionicpup32-8.0-uefi - not
